sample-lnd.conf

```diff
@@ -305,13 +305,27 @@
 ; The default value below is 20 MB (1024 * 1024 * 20)
 ; blockcachesize=20971520
 
+; DEPRECATED: Use 'fee.url' option. Optional URL for external fee estimation.
+; If no URL is specified, the method for fee estimation will depend on the
+; chosen backend and network. Must be set for neutrino on mainnet.
+; Default:
+;   feeurl=
+; Example:
+;   feeurl=https://nodes.lightning.computer/fees/v1/btc-fee-estimates.json
+
 ; Optional URL for external fee estimation. If no URL is specified, the method
 ; for fee estimation will depend on the chosen backend and network. Must be set
 ; for neutrino on mainnet.
 ; Default:
-;   feeurl=
+;   fee.url=
```

Comment:
Since the prefix creates a new sub config part, this whole section should get a section title (`[fee]`) and be moved to just above the `[prometheus]` section, otherwise this becomes syntactically invalid for the config file parser (even though people shouldn't use this as a template directly, a lot of people probably still will).